dask/array/core.py

```diff
@@ -3187,6 +3224,16 @@ def cov(m, y=None, rowvar=1, bias=0, ddof=None):
         return (dot(X, X.T.conj()) / fact).squeeze()
 
 
+@wraps(np.corrcoef)
+def corrcoef(x, y=None, rowvar=1):
+    c = cov(x, y, rowvar)
+    if c.shape == ():
+        return c/c
+    d = diag(c)
+    d = d.reshape((d.shape[0], 1))
+    return c / sqrt(d * d.T)
```

Comment:
I'm not against changing this, but I really don't think it does. Both operations are fully chunk aligned elementwise operations. The way I currently have it results in:

```
out[i, j] = c[i, j]/sqrt(d[i] * d[j].T)
```

The way you suggest it would be:

```
out[i, j] = c[i, j]/(sqrt(d[i]) * sqrt(d[j].T))
```

where `i` and `j` are the chunk indices. Each output cell draws on the same intermediate results. Thinking about it just from a key-value perspective, the x*x^T matrix is never actually constructed. I don't know the internals of the distributed scheduler as well, but I'd assume it would also fair fine in either situation? Correct me if I'm wrong here.